jekyll/_cci2/oss.adoc

```diff
@@ -40,21 +40,67 @@ For more information, see the xref:set-environment-variable#set-an-environment-v
 [#only-build-pull-requests]
 === Only build pull requests
 
-include::../_includes/partials/notes/standalone-unsupported.adoc[This feature is not supported for GitLab, GitHub App or Bitbucket Data Center]
+NOTE: This feature *is not supported* for orgs that use the Bitbucket Data Center integration.
```

Comment:
This looks brilliant! I think all that's missing is a v short intro about what the feature actually is and why use it